README.md

```diff
@@ -1 +1,111 @@
+[![GCC](https://github.com/maliput/maliput_viz/actions/workflows/build.yml/badge.svg)](https://github.com/maliput/maliput_viz/actions/workflows/build.yml)
+
 # maliput_viz
+
+## Description
+
+`maliput_viz` package is part of the [Maliput](https://github.com/maliput/) family.
+It offers a visualizer for analyzing the road networks.
+
+**Note**: For full information about Maliput please visit [Maliput Documentation](https://maliput.readthedocs.io/en/latest/index.html).
+
+<img src="docs/maliput_viz.png">
+
+## How to use it
+
+`maliput_viz` application is provided by `maliput_viz` package.
+
+After installation simply run:
+```sh
+maliput_viz
+```
+
+In the top-right of the window options about the road network loading can be found:
+
+<img src="docs/road_network_loader.png">
+
+`maliput_viz` uses the [maliput plugin architecture](https://maliput.readthedocs.io/en/latest/html/deps/maliput/html/maliput_plugin_architecture.html) for loading the maliput backends in runtime.
+
+In general, each backend would receive a set of parameter that defines the behavior of the road network loader. For example, for the `maliput_malidrive` backend, the `opendrive_file` parameter holds the path to a `XODR` map file.
+
+When a backend is selected, the default parameter for that particular backend are filled up.
+The user can add, edit or remove any parameter as they please.
+
+After hitting the `Load Road Network` button the correspondant backend will load up a maliput RoadNetwork and it will be displayed.
```

Comment:
```suggestion
After hitting the `Load Road Network` button, the correspondent backend will load up a maliput `RoadNetwork` and it will be displayed.
```